src/Logging/PsrLogger.php

```diff
@@ -0,0 +1,235 @@
+     * @param array $context Please see {@see Google\Cloud\Logging\PsrLogger::log}
+     *        for the available options.
+     */
+    public function critical($message, array $context = [])
+    {
+        $this->log(Logger::CRITICAL, $message, $context);
+    }
+
+    /**
+     * Log an error entry.
+     *
+     * Example:
+     * ```
+     * $psrLogger->error('error message');
+     * ```
+     *
+     * @param string $message The message to log.
+     * @param array $context Please see {@see Google\Cloud\Logging\PsrLogger::log}
+     *        for the available options.
+     */
+    public function error($message, array $context = [])
+    {
+        $this->log(Logger::ERROR, $message, $context);
+    }
+
+    /**
+     * Log a warning entry.
+     *
+     * Example:
+     * ```
+     * $psrLogger->warning('warning message');
+     * ```
+     *
+     * @param string $message The message to log.
+     * @param array $context Please see {@see Google\Cloud\Logging\PsrLogger::log}
+     *        for the available options.
+     */
+    public function warning($message, array $context = [])
+    {
+        $this->log(Logger::WARNING, $message, $context);
+    }
+
+    /**
+     * Log a notice entry.
+     *
+     * Example:
+     * ```
+     * $psrLogger->notice('notice message');
+     * ```
+     *
+     * @param string $message The message to log.
+     * @param array $context Please see {@see Google\Cloud\Logging\PsrLogger::log}
+     *        for the available options.
+     */
+    public function notice($message, array $context = [])
+    {
+        $this->log(Logger::NOTICE, $message, $context);
+    }
+
+    /**
+     * Log an info entry.
+     *
+     * Example:
+     * ```
+     * $psrLogger->info('info message');
+     * ```
+     *
+     * @param string $message The message to log.
+     * @param array $context Please see {@see Google\Cloud\Logging\PsrLogger::log}
+     *        for the available options.
+     */
+    public function info($message, array $context = [])
+    {
+        $this->log(Logger::INFO, $message, $context);
+    }
+
+    /**
+     * Log a debug entry.
+     *
+     * Example:
+     * ```
+     * $psrLogger->debug('debug message');
+     * ```
+     *
+     * @param string $message The message to log.
+     * @param array $context Please see {@see Google\Cloud\Logging\PsrLogger::log}
+     *        for the available options.
+     */
+    public function debug($message, array $context = [])
+    {
+        $this->log(Logger::DEBUG, $message, $context);
+    }
+
+    /**
+     * Write a log entry.
+     *
+     * Example:
+     * ```
+     * use Google\Cloud\Logging\Logger;
+     *
+     * $psrLogger->log(Logger::ALERT, 'alert message');
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @param string $level The severity of the log entry.
+     * @param string $message The message to log.
+     * @param array $context {
+     *     @type array $httpRequest Information about the HTTP request
```

Comment:
We talked a while ago about providing serializers for the most popular HTTP request wrappers (symfony, psr-7, etc). Do you still think that's valuable? I don't want to worry about it now, but it might be something to keep in our back pocket for later.